tests/aslts/src/runtime/cntl/common.asl

```diff
@@ -795,13 +813,14 @@ Method(err, 7)
 
 /*
  * Report parameters of error
- * arg0 - absolute index of file reporting the error
+ * arg0 - absolute index of file reporting the error and line number
  * arg1 - index of error
  * arg2 - absolute index of file initiating the checking
  * arg3 - name of Method initiating the checking
  * arg4 - index of checking
+ * arg5 - line number of error
```

Comment:
I mean - we can directly convert arg4 or arg1 into __LINE__, there is no need to introduce arg5 here.